func.go

```diff
@@ -432,8 +432,8 @@ func (c *FuncCmd) SetEnv(env []string) {
 		nameVal := strings.Split(e, "=")
 		name := nameVal[0]
 		var val string
-		if len(nameVal) == 1 {
-			val = nameVal[1]
+		if len(nameVal) >= 2 {
+			val = strings.Join(nameVal[1:], "=")
```

Comment:
Rather than this, just use `strings.SplitN(e, "=", 2)` on line 432. That way you only ever get at max 2 items in the split